samples/DotNet/Microsoft.Azure.ServiceBus/ServiceBusMessagingSamples.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27004.2005
+# Visual Studio Version 16
```

Comment:
Don't think it will matter, but customers that have not yet on VS2019 will be able to open the solution?

Reply:
They can use Visual Studio Code instead.

Reply:
I don't think it will matter because VS2017 will edit automatically the sln file.